wazimap/data/tables.py

```diff
@@ -310,7 +317,7 @@ class FieldTable(SimpleTable):
 
     """
     def __init__(self, fields, id=None, universe='Population', description=None, denominator_key=None,
-                 table_per_level=False, has_total=True, **kwargs):
+                 table_per_level=False, has_total=True, total_type='Integer', **kwargs):
```

Comment:
Can we make this ``value_type`` throughout instead? Linking it to the total seems weird for tables for which we can specify ``has_total=False``.